podoglyph/gradebook_v
# Development

#### What does this PR do?
This PR works through the teacher_can_view_courses_semester_spec. It creates migrations, models, and model_specs for Course, Semester, and SemesterCourses and tests their associations.

Also, add user login info to Readme.

Add Courses, Semesters, and SemesterCourses to seed file.

#### How should this be manually tested?
You can login as a Teacher and see the dashboard view with data. The TeacherCourse view is created, but without data at this time.

#### Any background context you want to provide?

#### Questions:

  - Do Migrations Need to be ran?
* Yes
  - Do Environment Variables need to be set?
* No
  - Any other deploy steps?
* No
